vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -19,6 +19,8 @@
 
 __all__ = ['ImagePredictor']
 
+from ...core.utils.multiprocessing_utils import is_fork_enabled
```

Comment:
shall we use relative import or from autogluon.core?

Reply:
absolute import should be used

Reply:
fixed